libs/docs/cdk/breakpoint/breakpoint-header/breakpoint-header.component.html

```diff
@@ -0,0 +1,12 @@
+<header>Breakpoint</header>
+<description>
+    <p>
+        During the development of a web application or a component, we often need a logic for determining visibility of
```

Comment:
I think the docs need to give some more detail for when using the default values, i.e. pixel amounts, whether the breakpoints work as minimums or maximums, etc